tools/eslint-rules/tsconfig.json

```diff
@@ -1,7 +1,11 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "module": "ESNext"
+    "module": "ESNext",
+    "moduleResolution": "NodeNext",
```

Comment:
types from ts-eslint utils didnt work. introduced in https://github.com/microsoft/fluentui/pull/31988/files#diff-fdbad6033386936e99fb9a734633e0abd3dfd71477c9cd8ff9564ef5ede613f0 but didnt fail on CI as we were nut having `type-check` for eslint-rules project.